beacon-chain/rpc/beacon/validators.go

```diff
@@ -186,8 +186,11 @@ func (bs *Server) ListValidators(
 		}
 		requestedEpoch = q.Epoch
 	}
+	reqState, err := bs.HeadFetcher.HeadState(ctx)
```

Comment:
only fetch the head state if
```
	if requestedEpoch == currentEpoch {
```
otherwise it is a wasted copy

Reply:
fixed